src/vaults/mainnet.json

```diff
@@ -10,6 +10,11 @@
       "slug": "beraborrow-bwbera",
       "stakeTokenAddress": "0x6751C09113a83a83a43829fD0b3BC0D7bdbe07bf"
     },
+    {
+      "beraRewardsVault": "0x597877Ccf65be938BD214C4c46907669e3E62128",
+      "slug": "beraborrow-snect",
+      "stakeTokenAddress": "0x1cE0a25D13CE4d52071aE7e02Cf1F6606F4C79d3"
```

Comment:
```suggestion
      "stakeTokenAddress": "0x597877Ccf65be938BD214C4c46907669e3E62128"
```

Reply:
this should be staked nectar token.
and [`0x1cE0a25D13CE4d52071aE7e02Cf1F6606F4C79d3`](https://berascan.com/address/0x1cE0a25D13CE4d52071aE7e02Cf1F6606F4C79d3) is nectar which should be the underlying token